docs/_layouts/homepage.html

```diff
@@ -6,7 +6,7 @@
         {% include generic-content.html %}
     </div>
     <div class="o-editor_link" id="edit-page">
-        <a class="a-btn" href="{{ site.baseurl }}/admin/#/collections/generic-pages/entries/home" title="Edit this page in Netlify CMS">
+        <a class="a-btn" href="{{ "/admin/#/collections/generic-pages/entries/home" | relative_url }} title="Edit this page in Netlify CMS">
```

Comment:
Missing the closing `"` at the end of the `href`:

```suggestion
        <a class="a-btn" href="{{ "/admin/#/collections/generic-pages/entries/home" | relative_url }}" title="Edit this page in Netlify CMS">
```

Reply:
Let's single quote it inside for clarity?